sips/sip-031/sip-031.md

```diff
@@ -0,0 +1,572 @@
+2. Increasing usage of apps where users pay gas in sBTC drives up protocol transactions — each of which ultimately requires STX at the protocol layer.  
+3. Similarly, enabling DeFi vaults with native liquidity pools ensures deeper markets for Stacks DeFi, which increases sBTC utility, boosts protocol revenue, and creates sustainable STX buy pressure while bringing new users and capital into the Stacks ecosystem to engage with the entire universe of Bitcoin apps built here.
+
+In each case, targeted capital deployment turns ecosystem growth into protocol-level demand for STX.
+
+## Key SIP Design Principles
+
+In putting this SIP together, there were a few clear guiding principles:
+
+1. **Continued commitment to meaningful decentralization.** This proposal is designed to uphold and enhance decentralization, while enabling more efficient day-to-day operations.  
+2. **Adapting to growth.** Stacks has evolved well beyond its early (Blockstack days) origins, and the mission has expanded significantly thanks to the ecosystem’s success. This SIP aims to align with the current landscape and increased opportunities, recognizing that capital needs have grown alongside this broader scope.  
+3. **Empowering builders.** The SIP should ensure that all Stacks builders, small & large, have access to strategic capital, resources, and execution they need for their given stage and focus. This support is a priority-unlocking faster go-to-market and scaling for high-potential projects within the ecosystem.  
+4. **Strengthening accountability.** Mechanisms for community oversight of new entities are either maintained or improved, reflecting direct feedback from the community.  
+5. **Balancing short- and long-term needs.** The SIP should address immediate priorities and sustainable, long-term capital required to grow the project for many decades.
+
+## Maintaining Strong Decentralization
+
+The commitment to decentralization remains. Let’s quickly reflect on the questions originally asked when forming the current ecosystem: [https://blog.blockstack.org/the-stacks-ecosystem-next-steps-on-the-path-to-decentralization](https://blog.blockstack.org/the-stacks-ecosystem-next-steps-on-the-path-to-decentralization).
+
+* **Technology**: To what extent is the technology itself centralized? Are there parts of the technology that are owned or controlled by particular individuals or entities?  
+* **Governance:** To what extent can an individual or single entity dictate changes to the technology and decide its future?  
+* **Ecosystem:** To what extent do single individuals or entities hold such influence that there is little room for contribution from others?
+
+The authors of this SIP are committed to maintaining decentralization in alignment with these established values. This proposal identifies opportunities to streamline key ecosystem operations while preserving essential checks and balances, ensuring the continued benefits of decentralization.
```

Comment:
i don't think this list or the preceding sentence add any value here - the last paragraph is what's important, if this SIP is concerned with appearing to centrailze.